src/ToolBox/SOS/Strike/util.cpp

```diff
@@ -4138,7 +4155,40 @@ void ResetGlobals(void)
 
 HRESULT LoadClrDebugDll(void)
 {
-#ifndef FEATURE_PAL
+#ifdef FEATURE_PAL
+    if (!g_palInitialized)
+    {
+        int err = PAL_Initialize(0, NULL);
+        if(err != 0)
+        {
+            return E_FAIL;
+        }
+        g_palInitialized = TRUE;
+    }
+    if (g_clrData == NULL)
+    {
+        HMODULE hdac = LoadLibraryA("/ssd/coreclr/binaries/Product/amd64/debug/" MAKEDLLNAME_A("mscordaccore"));
```

Comment:
I know this is temporary but let's not hardcode any path in here. Let's just leave it as "MAKEDLLNAME_A("mscordaccore")". You can set environment variable LD_LIBRARY_PATH (on Linux) or DYLD_LIBRARY_PATH (on Mac) to help the OS loader find the library. You can actually add this information to the error message for the case when dlopen fails.